packages/template-retail-react-app/app/pages/account/profile.jsx

```diff
@@ -232,11 +232,9 @@ const PasswordCard = () => {
     const {formatMessage} = useIntl()
     const headingRef = useRef(null)
     const {data: customer} = useCurrentCustomer()
-    const {isRegistered, customerId, email} = customer
-
-    const login = useAuthHelper(AuthHelpers.LoginRegisteredUserB2C)
+    const {isRegistered} = customer
 
-    const updateCustomerPassword = useShopperCustomersMutation('updateCustomerPassword')
+    const updateCustomerPassword = useAuthHelper(AuthHelpers.UpdateCustomerPassword)
```

Comment:
From a naming perspective it would be hard to understand the difference between using the mutation `useShopperCustomersMutation('updateCustomerPassword')` and the helper `useAuthHelper(AuthHelpers.UpdateCustomerPassword)` purely from those names we've chosen to use. 

We could consider renaming or adding a comment here to say why we are using the helper and mention that it will re-authenticate the user. The later might be the easier thing to do

Reply:
I've added a comment to this section explaining why we are using `AuthHelpers.UpdateCustomerPassword`